reflex/app.py

```diff
@@ -187,8 +193,6 @@ def __init__(self, *args, **kwargs):
             # 2 substates are built-in and not considered when determining if app is stateless.
             if len(State.class_subclasses) > 2:
                 self.state = State
```

Comment:
I think this line probably needs to go, otherwise it's still enabling the state based on number of subclasses still.

Reply:
Indeed, I removed those lines and added an unused State to the tailwind integration test